SECURITY.md

```diff
@@ -8,11 +8,10 @@
 
 Starting with release v4.6.0-rc.4, an SBOM is generated for each release. The [GitHub Actions release workflow](https://github.com/ICTU/quality-time/actions/workflows/release.yml) creates an Software Bill of Materials (SBOM) for the release, which can be found under the "Artifacts" header of the workflow run.
 
-## Supported Versions
+## Supported versions
 
 Only the latest version of *Quality-time* is currently being supported with security updates.
 
-## Reporting a Vulnerability
-
-Please report security vulnerabilities by email to the *Quality-time* team at quality-time@ictu.nl. The aim is to get back to you within 24 hours with a confirmation of the issue and a brief action plan or a request for more information.
+## Reporting a vulnerability
 
+Please report security vulnerabilities by [opening a draft security advisory](https://github.com/ICTU/quality-time/security/advisories/new).
```

Comment:
I'm not sure that a user without access would see this as "a draft security advisory", the button on the security page says "Report a vulnerability" (when not logged in).
For the other repos, I'm using `You can privately [report a vulnerability issue in this repository's issue tracker](https://github.com/ICTU/ (repo) /security/advisories/new).`

Reply:
Sounds good, I have used your text.